js_modules/dagster-ui/packages/ui-core/src/assets/insights/AssetCatalogInsightsLineChart.tsx

```diff
@@ -195,6 +207,41 @@ export const AssetCatalogInsightsLineChart = React.memo(
       }),
       [renderTooltipFn, rgbColors],
     );
+
+    const chartRef = useRef(null);
+    const onClick = (event: any) => {
+      if (chartRef.current) {
```

Comment:
Return early if false instead, since the entire operation of the function is within the conditional?